delta/plugins/elasticsearch/src/main/resources/defaults/default-settings.json

```diff
@@ -0,0 +1,38 @@
+{
+  "analysis": {
+    "analyzer": {
+      "nexus": {
+        "type": "custom",
+        "tokenizer": "classic",
+        "filter": [
+          "my_multiplexer"
+        ]
+      }
+    },
+    "filter": {
+      "my_multiplexer": {
+        "type": "multiplexer",
+        "filters": [
+          "uuid_type_4",
+          "word_delimiter_nexus",
+          "lowercase"
+        ]
+      },
+      "uuid_type_4": {
+        "type": "pattern_capture",
+        "preserve_original": true,
+        "patterns": [
+          "([a-f0-9]{8})\\-([a-f0-9]{4})\\-([a-f0-9]{4})\\-([a-f0-9]{4})\\-([a-f0-9]{12})"
+        ]
+      },
+      "word_delimiter_nexus": {
+        "type": "word_delimiter",
+        "split_on_numerics": false
+      }
+    }
+  },
+  "index": {
+    "number_of_shards": 1,
+    "number_of_replicas": 0
```

Comment:
We will need replicas for the index

Reply:
This is the minimum configuration suitable which will work best for the tests. I've created #2052 to address that separately.